### PR TITLE
Milestone 2

### PR DESCRIPTION
**Pull request by rociac**
This branch contains all the main components, the initial Redux store, actions and reducers.

### CODE REVIEW

File: src/components/BooksForm.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+const categories = [{ id: 1, category: 'Action' },
+  { id: 2, category: 'Biography' },
+  { id: 3, category: 'History' },
+  { id: 4, category: 'Horror' },
+  { id: 5, category: 'Kids' },
+  { id: 6, category: 'Learning' },
+  { id: 7, category: 'Sci-Fi' }];
+
```
Comment (shloch) [inline src/components/BooksForm.js]: this component should be placed in  `src/containers` directory

File: src/components/BooksList.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Book from './Book';
+
+const BooksList = ({ books }) => {
+  const booksArr = books.map(book => (
+    <Book book={book} key={book.id} />
+  ));
```
Comment (shloch) [inline src/components/BooksList.js]: this component should be placed in  `src/containers` directory

**Review by shloch — CHANGES_REQUESTED** [review]
## great job @rociac , @Juakata 

AWESOME JOB
### PLEASE RELOCATE 2 FILES MENTIONED BELOW AND FILE FOR LAST REVIEW, thats all you have to do guys.......... :)

**Comment by rociac** [comment]
Made the requested changes :)

**Review by bellom — APPROVED** [review]
## Hi Team :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**